client/components/web-preview/content.jsx

```diff
@@ -112,13 +112,33 @@ export class WebPreviewContent extends Component {
 			case 'location-change':
 				this.handleLocationChange( data.payload );
 				return;
+			case 'focus':
+				this.removeSelection();
+				return;
 		}
 	}
 
 	handleLocationChange = ( payload ) => {
 		this.props.onLocationUpdate( payload.pathname );
 	}
 
+	removeSelection = () => {
+		// remove all textual selections when user gives focus to preview iframe
+		// they might be confusing
+		if ( global.window ) {
+			if ( typeof window.getSelection !== 'undefined' ) {
```

Comment:
What do you think to making use of lodash's utils here? We could make this chunk a lot more readable, at least in my eyes:
```js
if ( isUndefined( window.getSelection ) ) { }
...
if ( isFunction( selection.empty ) ) {
     selection.empty();
} else if ( isFunction( selection.removeAllRanges ) ) {
     selection.removeAllRanges();
}
```